res/controllers/Keyboard-Default.kbd.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- TODO(Tomasito): Change this to the version in which KeyboardController support will be added-->
+<MixxxKeyboardPreset schemaVersion="1" mixxxVersion="2.0.0+">
+  <info>
+    <name>Keyboard Default</name>
+    <author>Mixxx</author>
+    <description>Default keyboard mapping. (Layout: en_US)</description>
+    <wiki>https://www.mixxx.org/manual/latest/chapters/appendix.html#appendix-keyboard</wiki>
+  </info>
+
+  <controller>
+    <group name="[AutoDJ]">
```

Comment:
I did have a look at the midi controller XML format, [and I even tried it out](https://github.com/Tomasito665/mixxx/commit/df0ae4d70e07c577ac445e81e095c989483a6e91#commitcomment-17835304). But only for the first group I ended up going from 11 lines of code in the `kbd.cfg` format to 23 lines of code in a modified version of the midi controller XML format.

**kbd.cfg format:**
```cfg
[AutoDJ]
shuffle_playlist Shift+F9
skip_next Shift+F10
fade_now Shift+F11
enabled Shift+F12
```

**Modified midi controller format:**
```xml
<control>
  <group>[AutoDJ]</group>
  <action>shuffle_playlist</action>
  <keyseq>Shift+F9</keyseq>
</control>

<control>
  <group>[AutoDJ]</group>
  <action>skip_next</action>
  <keyseq>Shift+F10</keyseq>
</control>

<control>
  <group>[AutoDJ]</group>
  <action>fade_now</action>
  <keyseq>Shift+F11</keyseq>
</control>

<control>
  <group>[AutoDJ]</group>
  <action>enabled</action>
  <keyseq>Shift+F12</keyseq>
</control>
```

And of course, the midi controller XML format also includes `<status>`, `<midino>` and `<options>` elements. If we need that information for keyboard controllers as well it might be better to adopt the midi format, though I doubt that we will need `<status>` and `<midino>`. It's an other story for `<options>`. That would probably do make sense to adopt, although I don't know if we would need to support [all input options](http://www.mixxx.org/wiki/doku.php/midi_controller_mapping_file_format#input_options). In fact, I would say we only need: `script-binding`, `button` and `toggle`, but I haven't yet fully dived into all the capabilities of midi controllers yet, so I am not so sure yet, what do you think?